src/sales-channels/asc/ready-to-list.md

```diff
@@ -3,29 +3,23 @@ title: Ready to List
 ---
 
 
-The _Ready to List_ tab shows your Magento catalog meet your defined listing settings and are ready to publish to Amazon as a **new** listing. This tab appears when you click **Manage Listings** on the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
+The _Ready to List_ tab shows your Magento catalog meet your defined listing settings and are ready to publish to Amazon as a **new** listing. This tab appears on the _Product Listings_ page when you click **Manage Listings** on the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
```

Comment:
?? This needs and edit but not sure what you are trying to say here.

Reply:
corrected in next commit